### PR TITLE
add messages_password to variables

### PR DESCRIPTION
**Pull request by 2019jack**
* feat: messages_password use variables

### CODE REVIEW

**Comment by br3d** [comment]
Hello @2019jack 
Thank you for your PR. Could you replace the double underscore on the single? To match the rest style of the variables writing.

**Comment by 2019jack** [comment]
> Hello @2019jack Thank you for your PR. Could you replace the double underscore on the single? To match the rest style of the variables writing.

maybe I didn't understand what you mean, can you point it out specifically?

File: ansible/inventories/mainnet.yml [hunk 1]
```diff
@@ -31,6 +31,7 @@ validators_ejector:
       - "http://{{ (hostvars[groups.ethereum_nodes[0]]).wireguard_address.split('/')[0] }}:5051"
     lido_validator_ejector_operator_id: "{{ lookup('env','VALIDATOR_EJECTOR_OPERATOR_ID') }}"
     lido_validator_ejector_staking_module_id: "{{ lookup('env','VALIDATOR_EJECTOR_STAKING_MODULE_ID') }}"
+    lido_validator_ejector_messages_password: "{{ lookup('env','VALIDATOR_EJECTOR__MESSAGES_PASSWORD') }}"
```
Comment (br3d) [inline ansible/inventories/mainnet.yml]: Could you replace the double underscore on the single? To match the rest style of the variables writing.
VALIDATOR_EJECTOR__MESSAGES_PASSWORD -> VALIDATOR_EJECTOR_MESSAGES_PASSWORD

File: ansible/inventories/testnet.yml [hunk 2]
```diff
@@ -37,6 +37,7 @@ validators_ejector:
       - "http://{{ (hostvars[groups.ethereum_nodes[0]]).wireguard_address.split('/')[0] }}:5051"
     lido_validator_ejector_operator_id: "{{ lookup('env','VALIDATOR_EJECTOR_OPERATOR_ID') }}"
     lido_validator_ejector_staking_module_id: "{{ lookup('env','VALIDATOR_EJECTOR_STAKING_MODULE_ID') }}"
+    lido_validator_ejector_messages_password: "{{ lookup('env','VALIDATOR_EJECTOR__MESSAGES_PASSWORD') }}"
```
Comment (br3d) [inline ansible/inventories/testnet.yml]: Could you replace the double underscore on the single? To match the rest style of the variables writing.
VALIDATOR_EJECTOR__MESSAGES_PASSWORD -> VALIDATOR_EJECTOR_MESSAGES_PASSWORD

File: base.sh [hunk 3]
```diff
@@ -69,6 +69,7 @@ export_custom_envs() {
   export VALIDATOR_EJECTOR_LOCATOR_ADDRESS=${VALIDATOR_EJECTOR_LOCATOR_ADDRESS}
   export VALIDATOR_EJECTOR_STAKING_MODULE_ID=${VALIDATOR_EJECTOR_STAKING_MODULE_ID}
   export VALIDATOR_EJECTOR_OPERATOR_ID=${VALIDATOR_EJECTOR_OPERATOR_ID}
+  export VALIDATOR_EJECTOR__MESSAGES_PASSWORD=${VALIDATOR_EJECTOR__MESSAGES_PASSWORD}
```
Comment (br3d) [inline base.sh]: Could you replace the double underscore on the single? To match the rest style of the variables writing.
VALIDATOR_EJECTOR__MESSAGES_PASSWORD -> VALIDATOR_EJECTOR_MESSAGES_PASSWORD

File: variables [hunk 4]
```diff
@@ -12,3 +12,4 @@ TELEGRAM_CHAT_ID=-123456
 VALIDATOR_EJECTOR_LOCATOR_ADDRESS=0x1eDf09b5023DC86737b59dE68a8130De878984f5 # Goerli contract. No mainnet contract exists currently.
 VALIDATOR_EJECTOR_STAKING_MODULE_ID=0
 VALIDATOR_EJECTOR_OPERATOR_ID=0
+VALIDATOR_EJECTOR__MESSAGES_PASSWORD=pass
```
Comment (br3d) [inline variables]: Could you replace the double underscore on the single? To match the rest style of the variables writing.
VALIDATOR_EJECTOR__MESSAGES_PASSWORD -> VALIDATOR_EJECTOR_MESSAGES_PASSWORD

Comment (2019jack) [reply variables]: > Could you replace the double underscore on the single? To match the rest style of the variables writing. VALIDATOR_EJECTOR__MESSAGES_PASSWORD -> VALIDATOR_EJECTOR_MESSAGES_PASSWORD

got it! sorry

File: base.sh (hunk 3 above)
Comment (2019jack) [reply base.sh]: > Could you replace the double underscore on the single? To match the rest style of the variables writing. VALIDATOR_EJECTOR__MESSAGES_PASSWORD -> VALIDATOR_EJECTOR_MESSAGES_PASSWORD

updated

**Pull request merged**